modules/rosa-deleting-account-wide-iam-roles-and-policies.adoc

```diff
@@ -20,7 +20,7 @@ The account-wide IAM roles and policies might be used by other ROSA clusters in
 .Prerequisites
 
 * You have installed a ROSA cluster.
-* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
+* You have installed and configured the latest {product-title} (ROSA) CLI (`rosa`) on your installation host.
```

Comment:
"{product-title} (ROSA)" should be used on the first mention in a section, so probably the first paragraph of this module, and then this can just be "ROSA CLI (`rosa`)"

Reply:
"{product-title} (ROSA)" is already mention in first paragraph. So, I will update to "ROSA CLI (`rosa`)" as you suggested. Thank you!